docs/templating.md

```diff
@@ -68,7 +68,7 @@ For OpenAPI Generator versions prior to 5.0, you will want to find the [resource
 embeddedTemplateDir = templateDir = "csharp-netcore";
 ```
 
-These templates are in our source repository at [modules/openapi-generator/src/main/resources/csharp-netcore](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/csharp-netcore). Be sure to select the tag or branch for the version of OpenAPI Generator you're using before grabbing the templates.
```

Comment:
what about renaming csharp-netcore to csharp instead?

Reply:
done.